### PR TITLE
feat(halogen): definitions

### PR DESCRIPTION
**Pull request by steller**
New definition type for a react component halogen

### CODE REVIEW

File: halogen/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+// Project: https://github.com/yuanyan/halogen
+// Definitions by: Clément Devos <https://github.com/steller>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "halogen" {
+  import * as react from "react";
+
+  type VerticalAlign = "baseline" | "length" | "sub" | "super" | "top" | "text-top" | "middle" | "bottom" | "text-bottom" | "initial" | "inherit";
+
+  interface IHalogenCommonProps {
```
Comment (andy-ms) [inline halogen/index.d.ts]: Don't need `I`

File: halogen/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    height?: string;
+    width?: string;
+    radius?: string;
+  }
+
+  /**
+   * React components
+   */
+  type PulseLoader = React.Component<IMarginLoaderProps<string>, {}>;
+  export let PulseLoader: React.ComponentClass<IMarginLoaderProps<string>>;
```
Comment (andy-ms) [inline halogen/index.d.ts]: Is this mutable? Otherwise use `export const`.

File: halogen/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  export let MoonLoader: React.ComponentClass<ISizeLoaderProps>;
+
+  type RingLoader = React.Component<ISizeLoaderProps, {}>;
+  export let RingLoader: React.ComponentClass<ISizeLoaderProps>;
+
+  type BounceLoader = React.Component<ISizeLoaderProps, {}>;
+  export let BounceLoader: React.ComponentClass<ISizeLoaderProps>;
+
+  type SkewLoader = React.Component<ISizeLoaderProps, {}>;
+  export let SkewLoader: React.ComponentClass<ISizeLoaderProps>;
```
Comment (andy-ms) [inline halogen/index.d.ts]: IMO if these values are all exactly the same, you don't need a test file for every one of them. Just a test for SizeLoaderProps.

**Review by ghost — COMMENTED** [review]
Please include a `tslint.json` with `{ "extends": "../tslint.json" }` and run `npm run lint -- halogen`.

File: halogen/index.d.ts (hunk 2 above)
Comment (steller) [inline halogen/index.d.ts]: I modified this in the last commit

File: halogen/index.d.ts (hunk 1 above)
Comment (steller) [inline halogen/index.d.ts]: I remove the `I` for the interface.
For the tslint I added the file, but it's don't nessessary. If you run `npm run lint -- halogen` it works without tslint.json. What it's a reason to added this file ?

Comment (steller) [inline halogen/index.d.ts]: ok thanks for explanation

**Pull request merged**